readme.rst

```diff
@@ -33,11 +33,11 @@ Start the ``sphinx-autobuild`` documentation server::
 
 	make run
 
-The documentation will be available at http://127.0.0.1:8000.
+The documentation will be available at ``http://127.0.0.1:8000``.
```

Comment:
A hyperlink without backticks is usefully clickable - this will prevent that.

Reply:
The reason for the change was that I don't want to put `http` in the word list for the spellchecker. But I used only `127.0.0.1:8000` as the link text now, that works as well.